{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,79 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly. Feel free to remove any
+sections that won't be relevant to reviewers of your PR, including this
+one.
+
+Anything marked with `TODO:` is something that you, the developer, should
+do as part of the PR.
+
+### Testing
+- TODO: Include unit, integration, acceptance tests as appropriate
```

Comment:
For the automated tests, I think the evidence is in the code, not in the checkbox. However, for the manual tests, I think a checkbox is a good call.